### PR TITLE
update RandomCrop class code annotation; test=document_fix

### PR DESCRIPTION
**Pull request by liyongchao911**
### PR types
Others

### PR changes
Docs

### Describe
1. add fill and paddling_mode parameters annotation in the API annotation area. The annotation ref pad API.
2. replace numpy with paddle function for example data generation in the the code example
3. add code anchor name for the CN docs copy function
4. giving the code print result with annotation

### CODE REVIEW

File: python/paddle/vision/transforms/transforms.py [hunk 1]
```diff
@@ -1036,9 +1036,27 @@ class RandomCrop(BaseTransform):
...
+                   padding [1, 2, 3, 4] with 2 elements on both sides in reflect mode
+                   will result in [3, 2, 1, 2, 3, 4, 3, 2]
+
+            - symmetric: pads with reflection of image (repeating the last value on the edge)
+
+                     padding [1, 2, 3, 4] with 2 elements on both sides in symmetric mode
+                     will result in [2, 1, 1, 2, 3, 4, 4, 3]
         keys (list[str]|tuple[str], optional): Same as ``BaseTransform``. Default: None.
     
     Shape:
```
Comment (Ligoml) [inline python/paddle/vision/transforms/transforms.py]: shape后面不用加`:`

File: python/paddle/vision/transforms/transforms.py [hunk 2]
```diff
@@ -1044,12 +1044,30 @@ class RandomCrop(BaseTransform):
             made.
         padding (int|sequence|optional): Optional padding on each border
```
Comment (Ligoml) [inline python/paddle/vision/transforms/transforms.py]: padding (int|sequence, optional)

File: python/paddle/vision/transforms/transforms.py [hunk 3]
```diff
@@ -1044,12 +1044,30 @@ class RandomCrop(BaseTransform):
...
             of the image. If a sequence of length 4 is provided, it is used to pad left, 
-            top, right, bottom borders respectively. Default: 0.
+            top, right, bottom borders respectively. Default: None, without padding.
         pad_if_needed (boolean|optional): It will pad the image if smaller than the
```
Comment (Ligoml) [inline python/paddle/vision/transforms/transforms.py]: pad_if_needed (boolean, optional)

File: python/paddle/vision/transforms/functional.py [hunk 4]
```diff
@@ -380,6 +380,7 @@ def adjust_brightness(img, brightness_factor):
 
     Examples:
         .. code-block:: python
+           :name: code-example1
 
             import numpy as np
```
Comment (Ligoml) [inline python/paddle/vision/transforms/functional.py:385]: 建议换成使用paddle来创建输入

Comment (liyongchao911) [reply python/paddle/vision/transforms/functional.py:385]: 已尝试将输入图片更改成paddle.tensor格式，在dev版本中会报错。建议此处还是使用Img.from_array来产生

File: python/paddle/vision/transforms/transforms.py (hunk 3 above)
Comment (liyongchao911) [reply python/paddle/vision/transforms/transforms.py]: done！

File: python/paddle/vision/transforms/transforms.py (hunk 2 above)
Comment (liyongchao911) [reply python/paddle/vision/transforms/transforms.py]: done！

File: python/paddle/vision/transforms/transforms.py (hunk 1 above)
Comment (liyongchao911) [reply python/paddle/vision/transforms/transforms.py]: done！

**Review by Ligoml — APPROVED** [review]
LGTM for docs

**Review by TCChenlong — APPROVED** [review]

**Pull request merged**